src/phoenix/trace/llama_index/callback.py

```diff
@@ -292,6 +352,8 @@ def _add_spans_to_tracer(
     span_exceptions: List[SpanEvent] = []
     while parent_child_id_stack:
         parent_span_id, event_id = parent_child_id_stack.pop()
+        # We have to rely on the fact that `event_id` from llama-index is a UUID.
```

Comment:
I think this comment is useful but it would probably be more useful if you explain why you need to rely on it being unique - notably that spans get exported as they arrive and the tree is re-constructed on the collector

Reply:
very good point. i will update it